app/scripts/lib/country-telephone-info.js

```diff
@@ -60,15 +60,41 @@ define((require, exports, module) => {
     return (serverPhoneNumber) => format.replace(/\$\{serverPhoneNumber\}/, serverPhoneNumber);
   }
 
+  function hasPrefix (num, prefix) {
+    return num.indexOf(prefix) === 0;
+  }
+
+  function ensurePrefix (prefix) {
+    return function (num) {
+      if (hasPrefix(num, prefix)) {
+        return num;
+      }
+      return `${prefix}${num}`;
+    };
+  }
+
   module.exports = {
+    // Austria
+    // https://en.wikipedia.org/wiki/Telephone_numbers_in_Austria
+    AT: {
+      format: formatter('+43 ${serverPhoneNumber}'),
+      normalize: ensurePrefix('+43'),
+      pattern: /^(?:\+43)?\d{6,}$/,
+      prefix: '+43',
+      rolloutRate: 0.5
```

Comment:
@davismtl - I know we discussed rolling this out to 50% of people, while discussing with @rfk today he suggested we could do a soft-launch, set the rate to 0, ask some community members to test once we release, and then bump the number the following train.

I think that's a reasonable suggestion.

Reply:
I think that would be prudent.

What would be the best way to reach out to DE community members to test?

Reply:
> What would be the best way to reach out to DE community members to test?

I know a few Mozillians in Germany that I can ask for help. I might need help with Austria though.